flamedisx/lxe_blocks/detection.py

```diff
@@ -123,13 +133,19 @@ def _compute(self, data_tensor, ptensor,
 class DetectElectrons(DetectPhotonsOrElectrons):
     dimensions = ('electrons_produced', 'electrons_detected')
 
-    special_model_functions = ('electron_acceptance',)
+    special_model_functions = ('electron_acceptance', 'extraction_eff')
```

Comment:
Currently you have `extraction_eff` as a special model function, taking `electrons_produced` as a bonus argument. But electrons_produced is the number of electrons captured before impurities, the extraction efficiency shouldn't be able to see that. Could you make it a regular model function instead, like electrons_detected? Sorry, didn't spot this on the first pass.

Reply:
Indeed it is a special model function at the moment and indeed in theory it shouldn't depend on electrons produced, however the point for the PR is that I wanted to make the `electron_detection_efficiency` a function of electrons produced somehow, and this is the easiest way I could think of. I could add another function to make it more physical, something like `electron_quenching` or `electron_loss` that depends on `electrons_produced` before the drift, and make the `extraction_eff` a regular model function with observable dependence and keyword arguments.